lib/graphql/define/assign_argument.rb

```diff
@@ -2,16 +2,21 @@ module GraphQL
   module Define
     # Turn argument configs into a {GraphQL::Argument}.
     module AssignArgument
-      def self.call(target, name, type = nil, description = nil, default_value: nil, &block)
+      def self.call(target, name, type = nil, description = nil, **rest, &block)
         argument = if block_given?
           GraphQL::Argument.define(&block)
         else
           GraphQL::Argument.new
         end
+
+        if (rest.keys - [:default_value]) != []
+          raise ArgumentError.new("unknown keyword: #{rest.keys.first}")
```

Comment:
Good catch. Yeah absolutely, I meant to grab the first key of the subtraction array.

If we want to be _correct_, we may want to do pluralization:

```
irb(main):006:0> a(one: 1, two: 2, three: 3)
ArgumentError: unknown keywords: two, three
...
irb(main):007:0> a(one: 1, two: 2)
ArgumentError: unknown keyword: two
...
irb(main):008:0>
```